### PR TITLE
Dev/v6 qtum

### PR DESCRIPTION
**Pull request by snowwhitedev**

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Before starting, ensure you've completed the instructions in the top-level [READ
 - Install dependencies\
   `yarn`
 
-- Ensure that you've [`yarn link`ed](https://classic.yarnpkg.com/en/docs/cli/link/) all `@web3-react/*` packages, as well as `react` and `react-dom`, from `../packages/*` and `../node_modules/{react,react-dom}` respectively. This will ensure that changes you make to individual packages will be reflected immediately in the example (as long as you're running `yarn start` in the root).
+- Ensure that you've [`yarn link`ed](https://classic.yarnpkg.com/en/docs/cli/link/) all `@qweb3/*` packages, as well as `react` and `react-dom`, from `../packages/*` and `../node_modules/{react,react-dom}` respectively. This will ensure that changes you make to individual packages will be reflected immediately in the example (as long as you're running `yarn start` in the root).
```
Comment (ripply) [inline example/README.md]: qtum-web3-react

File: lerna.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "6.2.14",
+  "version": "0.0.4",
```
Comment (ripply) [inline lerna.json]: restore version

File: packages/core/package.json [hunk 3]
```diff
@@ -1,26 +1,26 @@
 {
-  "name": "@web3-react/core",
+  "name": "@qtum-web3-react/core",
   "publishConfig": {
     "access": "public"
   },
-  "version": "6.1.9",
-  "description": "A simple, maximally extensible, dependency minimized framework for building modern Ethereum dApps",
+  "version": "0.0.4",
```
Comment (ripply) [inline packages/core/package.json]: restore version

File: package.json [hunk 4]
```diff
@@ -1,32 +1,25 @@
...
-    "authereum",
-    "magic"
+    "qnekt"
   ],
-  "author": "Noah Zinsmeister <noahwz@gmail.com>",
+  "author": "<snowwhite91924@gmail.com>",
   "repository": {
     "type": "git",
-    "url": "git://github.com/NoahZinsmeister/web3-react.git"
+    "url": "git://github.com/qtumproject/web3-react"
```
Comment (ripply) [inline package.json]: missing .git

File: packages/injected-connector/package.json [hunk 5]
```diff
@@ -1,15 +1,15 @@
 {
-  "name": "@web3-react/injected-connector",
+  "name": "@qtum-web3-react/injected-connector",
   "publishConfig": {
     "access": "public"
   },
-  "version": "6.0.7",
-  "description": "A simple, maximally extensible, dependency minimized framework for building modern Ethereum dApps",
+  "version": "0.0.4",
```
Comment (ripply) [inline packages/injected-connector/package.json]: restore version

File: packages/injected-connector/package.json [hunk 6]
```diff
@@ -35,8 +35,8 @@
     "lint": "tsdx lint src"
   },
   "dependencies": {
-    "@web3-react/abstract-connector": "^6.0.7",
-    "@web3-react/types": "^6.0.7",
+    "@qtum-web3-react/abstract-connector": "^0.0.4",
+    "@qtum-web3-react/types": "^0.0.4",
```
Comment (ripply) [inline packages/injected-connector/package.json]: version

File: packages/injected-connector/src/declarations.ts [hunk 7]
```diff
@@ -1,12 +1,12 @@
-interface Ethereum {
+interface Qtum {
```
Comment (ripply) [inline packages/injected-connector/src/declarations.ts]: Leave this as Ethereum, less changes the better

File: packages/injected-connector/src/index.ts [hunk 8]
```diff
@@ -98,102 +98,103 @@ export class InjectedConnector extends AbstractConnector {
...
+      window.qtum.removeListener('chainChanged', this.handleChainChanged)
+      window.qtum.removeListener('accountsChanged', this.handleAccountsChanged)
+      window.qtum.removeListener('close', this.handleClose)
+      window.qtum.removeListener('networkChanged', this.handleNetworkChanged)
     }
   }
 
   public async isAuthorized(): Promise<boolean> {
-    if (!window.ethereum) {
+    console.log("Uselesss ==>");
```
Comment (ripply) [inline packages/injected-connector/src/index.ts]: remove

**Review by ripply — COMMENTED** [review]
squash your changes so its just one commit

File: example/pages/index.tsx [hunk 9]
```diff
@@ -251,6 +211,37 @@ function App() {
...
+
+
+    const txHash = await nftContract.mint(account, 3); // 3 should be changed
+    // succeeded transaction hash => https://testnet.qtum.info/tx/563111a96ba2bbdc634e7f978d5bc7140cd31758027ff3c7dd8f9f45fe335cab
+    console.log('tx Hash ==>', txHash.toString());
+
+    const receipt = await txHash.wait();
+
+    console.log('receipt ==>', receipt);
+    // const nftContract = new Contract(contractAddress, contractAbi, library.getSigner(account).connectUnchecked())
```
Comment (ripply) [inline example/pages/index.tsx]: what is this commented code?

File: example/pages/index.tsx [hunk 10]
```diff
@@ -251,6 +211,37 @@ function App() {
...
+  const getProviderOrSigner = (library: Web3Provider): Web3Provider | JsonRpcSigner => {
+    return account ? getSigner(library) : library
+  }
+
+
+  const onHandleTestNFT = async () => {
+    const contractAddress = TEST_NFT_ADDRESS;
+    const contractAbi = CONTRACT_ABIS.ERC721_TEST_ABI;
+
+    // const nftContract = new Contract(contractAddress, contractAbi, getProviderOrSigner(library))
```
Comment (ripply) [inline example/pages/index.tsx]: commented code?

File: lerna.json [hunk 11]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "6.2.14",
+  "version": "6.2.15",
```
Comment (ripply) [inline lerna.json]: why bump version?

**Review by ripply — CHANGES_REQUESTED** [review]

File: lerna.json (hunk 11 above)
Comment (snowwhitedev) [reply lerna.json]: I can not publish as 6.2.14 because there is tag v6.2.14 already due to fork of web3-react

Comment (ripply) [reply lerna.json]: we can delete the tag

**Pull request merged**